docs/hyper-params.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>you said combinations twice, why not calling it <code>max_combinations</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/791/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>